recipes/magics-python/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "magics-python" %}
+{% set pypi_name = "Magics" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: b852ddc368116f9a3a8d9e778e7d82eb2f086601c5ff3cc61e2ed2928812d498
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - numpy
+    - python
```

Comment:
It's slightly messy. There's already a magics feedstock for the magics c++ library. ECMWF want users to `conda install magics` and get both magics and magics-python. That means magics-python needs to be a dep of magics. We would have a circular dependency if magics is also a dep of magics-python.

It's also possible for users to build their own magics from source. In that case they would want to install magics-python without having magics bundled with it.

Reply:
This will still introduce a circular dependency from being in the test dependencies.  From a quick search it seems like this is a change for magics 4, if so I have a suggestion but it might not be practical:

1. Make a `magics-cpp` package that conflicts with `magics <=4.0.0`
2. `magics-python` has a runtime dependency on `magics-cpp`
3. `magics` becomes a metapackage, possibly pinned to an exact version of `magics-cpp` so that it is still possible to run `conda install magics=4.3.2`

> It's also possible for users to build their own magics from source. In that case they would want to install magics-python without having magics bundled with it.

It might make more sense to only support this for pip installations of the Python bindings.

Reply:
You're right of course.

Your suggestion is decent but I don't really have the time to implement it. I think the solution which ticks all of the boxes at ECMWF and it also straightforward is to have this package untested (given that it's already tested elsewhere) and have it as a runtime dep of magics.

I've removed the import test entirely and conda build appears to be happy with it. Does that sound like an acceptable way forward to you?

Reply:
That's fine for me, LGTM 👍